Traceback (most recent call last):
  File "main.py", line 5, in <module>
    print os.system(ls)
NameError: name 'ls' is not defined
